xcmds/fixrsdp/fixrsdp.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2012-2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This command copies the existing RSDP into the EBDA. This is because
+// LinuxBoot tends to be EFI booted, which places the RSDP outside of the
+// low 1M or the EBDA. If Linuxboot legacy boots the following operating systems,
+// they may not have a way to find the RSDP afterwards.
```

Comment:
Improve this explanation a bit. Probably want to mention kexec.

Reply:
Not sure how to improve the explanation. I put in kexec